ref/general/what-is-open-liberty.adoc

```diff
@@ -16,4 +16,24 @@
 
 Open Liberty is a lightweight Java runtime for building cloud-native applications and microservices.
 
-With Open Liberty, it's easy to add and remove modular features from the latest versions of Jakarta EE and Eclipse MicroProfile. This modular structure simplifies microservice development, so you run Just enough Application Server to support the features that your application needs. Furthermore, with Open Liberty zero migration architecture, you can upgrade to the latest version with minimal impact to your current applications and configurations. Open Liberty is compatible with the Jakarta EE 8 Full Platform and Web Profile specifications and with MicroProfile 3.0. Open Liberty is deployed on every major developer platform, including Docker, Kubernetes, and Cloud Foundry. For the latest updates about Open Liberty features and capabilities, see the link:https://openliberty.io/blog/[Open Liberty blog] and https://openliberty.io/docs/[Open Liberty docs].
+With Open Liberty, you can add and remove modular features from the latest versions of Jakarta EE and Eclipse MicroProfile. This modular structure simplifies microservice development, so you run Just enough Application Server to support the features that your application needs. Furthermore, with Open Liberty zero migration architecture, you can upgrade to the latest version with minimal impact to your current applications and configurations. Open Liberty is compatible with the Jakarta EE 8 Full Platform and Web Profile specifications and with MicroProfile 3.0. Open Liberty is deployed on every major developer platform, including Docker, Kubernetes, and Cloud Foundry. For the latest updates about Open Liberty features and capabilities, see the link:https://openliberty.io/blog/[Open Liberty blog] and https://openliberty.io/docs/[Open Liberty docs].
+
+Open Liberty offers developers the following benefits:
+* A lightweight edition that corresponds to the Java EE Web Profile specification.
+* Excellent development and production runtime environments for web applications.
+* A smaller footprint for faster download and startup, giving more development time and faster time to deployment.
+* Ease of packaging applications for deployment, including configuration.
+* The ability to run applications that are written for Open Liberty on WebSphere Application Server traditional.
+* The ability to extend the Open Liberty capabilities by adding custom features that use a product extension System Programming Interface (SPI).
+
+== Server configuration
+
+Open Liberty follows a principle of configuration by exception, which allows for succinct config files. The runtime environment operates from a set of built-in config default settings. You only specify config that overrides the default settings. For more information, see link:/docs/ref/config/#serverConfiguration.html[Server Configuration].
```

Comment:
Change,
"You only specify config that overrides the default settings. "
to,
"You specify only the config that overrides the default settings. "